src/main/steps/common/documents/upload/utils.ts

```diff
@@ -50,7 +50,7 @@ export const deleteDocument = async (req: AppRequest, res: Response): Promise<vo
   try {
     await client.deleteCitizenStatementDocument(query.documentId as string);
 
-    if (req.session.userCase && req.session.userCase.hasOwnProperty(uploadedFilesDataReference)) {
+    if (req.session.userCase.hasOwnProperty(uploadedFilesDataReference)) {
```

Comment:
req.session?.userCase?.uploadedFilesDataReference

Reply:
Hi @vivek-sekhar , we cannot remove hasOwnProperty, as uploadedFilesDataReference is a constant which hold applicantUploadFiles' or 'respondentUploadFiles' depending on party type.